### PR TITLE
fix: bank transactions in mobile + bookeeping view consistency

### PR DESCRIPTION
**Pull request by imkesin**
## Description

- For transactions in mobile view, only show categorization to self-serve clients
- On mobile web, downloading an attachment will not change the browser URL (due to `content-disposition: attachment`)

### Screenshot

Screenshot 2024-11-22 at 5 14 23 PM

### CODE REVIEW

File: src/api/layer/bankTransactions.ts [hunk 1]
```diff
@@ -120,15 +120,15 @@ export const listBankTransactionDocuments = get<{
   errors: unknown
 }>(
   ({ businessId, bankTransactionId }) =>
-    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents?content_disposition=INLINE`,
+    `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents?content_disposition=ATTACHMENT`,
```
Comment (imkesin) [inline src/api/layer/bankTransactions.ts:123]: I confirmed with @AndrewAcomb that the original behavior had no material impact in web browsers.

File: src/styles/file_thumb.scss
Comment (imkesin) [inline src/styles/file_thumb.scss:1]: The changes in this file are intended to fix an unrelated bug: very large file names can easily overflow.

**Review by doneel — APPROVED** [review]
This looks great Is Your belief that this solves everything MoeGo is asking for?

**Review by tomaszantas — APPROVED** [review]
LGTM!

**Pull request merged**